tools/snp.sh

```diff
@@ -646,11 +681,23 @@ set_acl_for_sev_device() {
   echo "${setfacl_command}" | sudo tee -a "${rc_local_file}" >/dev/null
 }
 
-build_and_install_amdsev() {
+build_and_install_amdsev() {  
+  
   local amdsev_branch="${1:-${AMDSEV_DEFAULT_BRANCH}}"
-
+  
+    if [ "$SVSM" = true ]; then
+       local amdsev_branch="${1:-${AMDSEV_DEFAULT_BRANCH}}"
+	   
+    else
+       AMDSEV_URL=${AMDSEV_SVSM_URL}
+       amdsev_branch="svsm-latest"
+	   # Create directory
+       
+    fi	
```

Comment:
Your logic seems backward here?

Reply:
@ramagali24 

I keep seeing in your code that you check to see if $SVSM is true, but then if it is true, you proceed to do the non-svsm process, is there a reason you are doing it this way?

Reply:
for example lines 528 -> 543

Reply:
Addressed redundant re-declaration of amdsev_branch inside the if block.